### PR TITLE
Workload for project

### PR DESCRIPTION
**Pull request by g4s8**
#808 workload changes:
- added new constructor to `VsWorkload` to get votes for particular project
- added `Agenda.job(Project)` method to find jobs for project, updated test
- `@todo` to add unit-test for  `VsWorkload`

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1051](https://github.com/zerocracy/farm/issues/1051) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1051](https://github.com/zerocracy/farm/issues/1051) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/1470836925); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewercom.zerocracy.Par@52fd8d5c

File: src/test/java/com/zerocracy/pmo/AgendaTest.java [hunk 1]
```diff
@@ -87,4 +89,19 @@ public void removesOrdersFromEmptyAgenda() throws Exception {
         agenda.removeAll();
         MatcherAssert.assertThat(agenda.jobs(), Matchers.emptyIterable());
     }
+
+    @Test
+    public void jobsForProject() throws Exception {
```
Comment (carlosmiranda) [inline src/test/java/com/zerocracy/pmo/AgendaTest.java]: @g4s8 let's call this `returnsJobsForProject`.

File: src/test/java/com/zerocracy/pmo/AgendaTest.java [hunk 2]
```diff
@@ -87,4 +89,19 @@ public void removesOrdersFromEmptyAgenda() throws Exception {
...
+        final FkProject prjone = new FkProject("FAKEPRJC1");
```
Comment (carlosmiranda) [inline src/test/java/com/zerocracy/pmo/AgendaTest.java]: @g4s8 let's call this `first` for clarity

File: src/test/java/com/zerocracy/pmo/AgendaTest.java [hunk 3]
```diff
@@ -87,4 +89,19 @@ public void removesOrdersFromEmptyAgenda() throws Exception {
...
+        final FkProject prjtwo = new FkProject("FAKEPRJC2");
```
Comment (carlosmiranda) [inline src/test/java/com/zerocracy/pmo/AgendaTest.java]: @g4s8 and likewise, this one let's call `second`

File: src/main/java/com/zerocracy/pm/staff/votes/VsWorkload.java [hunk 4]
```diff
@@ -36,6 +38,10 @@
  * @author Kirill (g4s8.public@gmail.com)
  * @version $Id$
  * @since 0.17
+ * @todo #808:30min VsWorkload test is missed, workload votes are not testes
```
Comment (carlosmiranda) [inline src/main/java/com/zerocracy/pm/staff/votes/VsWorkload.java]: @g4s8 looks like there's an "interesting" typo here xD

Should be:

> ...workload votes are not tested

**Comment by g4s8** [comment]
@carlosmiranda should be fixed in 37f7cc8, please look again

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1051](https://github.com/zerocracy/farm/issues/1051) is now out of scope

**Comment by ypshenychka** [comment]
@carlosmiranda According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

All issues found during code review are cosmetic. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by ypshenychka** [comment]
@carlosmiranda Thank you

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**